terraform/modules/data/variables.tf

```diff
@@ -71,3 +81,14 @@ variable "primary" {
   default     = true
   description = "Whether the data container will operate as primary in a Primary-Replica configuration."
 }
+
+
+variable "cluster_id" {
+  description = "The ID of this data container in the cluster"
+  default     = null
+}
+
+variable "cluster_size" {
+  description = "The size of the cluster, if in cluster mode. Can be either 3 or 5"
```

Comment:
How bad is it if a value other than `3` or `5`is used?  Depending on the severity, it might be a good idea to implement input variable validation using one of the workarounds here (most recent one using `locals` seems simple enough): https://github.com/hashicorp/terraform/issues/2847